doc/src/tutorial/001_adding_lalrpop.md

```diff
@@ -28,11 +28,13 @@ look something like:
 name = "calculator"
 version = "0.1.0"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
+edition = "2021"
 
 [build-dependencies] # <-- We added this and everything after!
 lalrpop = "0.19.9"
 
 [dependencies]
+lalrpop = "0.19.9"
 lalrpop-util = "0.19.9"
```

Comment:
```suggestion
lalrpop-util = { version = "0.19.9", features = "lexer" }
```

as suggested in the comments.

Reply:
@youknowone give this a quick check -- but you should be able to commit the suggestion, as well, if it looks right to you. Ideally we would go over the docs to clarify the role of this feature and add intructions to the "custom lexer" section for the way to remove it.

Reply:
It looks working. I found `features = ["lexer"]` is added to `lalrpop` instead of `lalrpop-utils`. I made a fix, but is it correct fix?

Reply:
Also, since 0.20 we probably should add a `unicode` in the list of features